### DISCUSSION TITLE
Linux: Pressing L-R or volume crashes the application every time

### DISCUSSION THREAD

**Discussion by praveengitsit** [audacity/audacity#3263] [Q&A]
I make a track (mono, stereo, any) and try to change the L-R ratio or the volume from the left nav. Just that simple task makes it crash. 

**Steps to reproduce the behavior:**
1. Go to the left panel of a track
2. Click on L-R ratio or the volume

**Screenshots**
![ezgif-2-69a4ab0160](https://user-images.githubusercontent.com/45666428/172498350-78b84450-eddc-4183-b184-c2064a621a1a.gif)

**Additional information (please complete the following information):**
 - OS: **Ubuntu 20.04.4 LTS**
 - Audacity version: Latest stable i.e. **audacity-linux-3.1.3-x86_64.AppImage**

**Additional context**
This issue does not arrive when I use the top menu to make changes to the track's left or right pan. But the issue still exists for audio volume. I can click the volume, scroll the volume, any change to it will crash the app. Audacity worked fine before, but has not worked since the upgrade a few months earlier.

**Comment by petersampsonaudacity** [comment]
Testing on W11 with 3.1.3 and latest alpha master for 3.2.0

I do not get this crash
a) clicking in the Gain slider ruler
b) moving the Gain slider
c) clicking in the Pan slider ruler
b) moving the Pan slider
all work fine.

**Comment by praveengitsit** [comment]
@petersampsonaudacity

Yes, this works perfectly in a Windows device. My issue was with my primary machine which, as I mentioned before, has **Ubuntu 20.04.4 LTS**.

If there is any more info that I can give from my end to help resolve this, do let me know. I'll be glad to help!
